bloom_nofos/nofos/api/api.py

```diff
@@ -0,0 +1,58 @@
+from ninja import NinjaAPI, Schema
+from ninja.security import HttpBearer
+from django.core.exceptions import ValidationError
+from .schemas import NofoSchema, ErrorSchema, SuccessSchema
+from nofos.models import Nofo
+from nofos.views import _build_nofo
+from django.conf import settings
+
+
+class BearerAuth(HttpBearer):
+    def authenticate(self, request, token):
+        if token and settings.API_TOKEN and token == settings.API_TOKEN:
+            return token
+        return None
+
+
+api = NinjaAPI(
+    auth=BearerAuth(),
+    urls_namespace="api",
+    docs_url="/docs",
+)
+
+
+@api.post("/nofo/import", response={201: SuccessSchema, 400: ErrorSchema})
```

Comment:
I should have mentioned this earlier, but let's follow REST conventions a little closer.

This means:

- Let's use a plural object noun for this `/api/nofos/{id}`
- POST route should use the same path, doesn't need 'import': `/api/nofos`
- Return a 201 response 👍  (we are doing this)
- Return the newly created object rather than a success message
- Include a `Location` header with the URL of the newly created resource (`Location: /api/nofo/123`

Reply:
updated.  changed to get and create method names to be more conventional as well